projects/components/src/table/table.component.ts

```diff
@@ -345,10 +352,12 @@ export class TableComponent
   }
 
   public onHeaderCellClick(columnConfig: TableColumnConfig): void {
-    this.updateSort({
-      column: columnConfig,
-      direction: this.getNextSortDirection(columnConfig.sort)
-    });
+    if (this.isColumnSortable(columnConfig)) {
```

Comment:
Is sortable not used anywhere any more? must be a regression. We should also change the styling for not sortable so it doesn't have a hover pointer.

Reply:
Right. The sortable was behaving as a no-op (_as in it was always sorting, irrespective of the value_)